### PR TITLE
Add Network screen

### PR DESCRIPTION
**Pull request by Motzart**

### CODE REVIEW

File: app/screens/network/Network.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      <WrapperWith2SideBars width={1000} height={500} header="NETWORK" headerIcon={network} subHeader={networkName}>
+        <Container>
+          <Message color={nodeIndicator.color}>{nodeIndicator.message}</Message>
+          <DetailsWrap>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Age</DetailsText>
+                <Tooltip width="250" text="tooltip age" />
+              </DetailsTextWrap>
+              <GrayText>{getFormattedTimestamp(node.genesisTime)}</GrayText>
```
Comment (IlyaVi) [inline app/screens/network/Network.js:109]: connect 'genesisTime' instead of whole 'node' object

File: app/screens/network/Network.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                <Tooltip width="250" text="tooltip Status" />
+              </DetailsTextWrap>
+              <GrayText>{this.renderSyncingStatus(status)}</GrayText>
+            </DetailsRow>
+            <DetailsRow>
+              <DetailsTextWrap>
+                <DetailsText>Current Layer</DetailsText>
+                <Tooltip width="250" text="tooltip Current Layer" />
+              </DetailsTextWrap>
+              <GrayText>{node.status.currentLayer}</GrayText>
```
Comment (IlyaVi) [inline app/screens/network/Network.js:123]: 'status' is connected to props already

File: app/screens/network/Network.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            <Tooltip width="250" text="tooltip CHANGE NETWORK" />
+            <Link onClick={this.openLogFile} text="BROWSE LOG FILE" style={{ marginLeft: '50px' }} />
+            <Tooltip width="250" text="tooltip BROWSE LOG FILE" />
+          </FooterWrap>
+        </Container>
+      </WrapperWith2SideBars>
+    );
+  }
+
+  renderSyncingStatus = (status) => {
```
Comment (IlyaVi) [inline app/screens/network/Network.js:158]: no need to pass as arg a prop 'status', take it from 'this.props'

File: app/screens/network/Network.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        <ProgressLabel>{this.getSyncLabelPercentage(status)}%</ProgressLabel>
+        <ProgressLabel>{this.getSyncIndicator(status)}</ProgressLabel>
+        <Progress>
+          <ProgressBar progress={progress} />
+        </Progress>
+      </>
+    );
+  };
+
+  renderStatus = (status) => (
```
Comment (IlyaVi) [inline app/screens/network/Network.js:174]: redundant func

File: app/screens/network/Network.js [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    <>
+      <NetworkIndicator status={status} />
+      <ProgressLabel>{getNodeStatusText(status)}</ProgressLabel>
+    </>
+  );
+
+  getSyncIndicator = (status) => {
```
Comment (IlyaVi) [inline app/screens/network/Network.js:181]: also redundant func

File: app/infra/utils.js [hunk 6]
```diff
@@ -57,4 +57,14 @@ const formatSmidge = (amount: number, separateResult) => {
   return separateResult ? { value: res.value, unit: res.unit } : `${res.value} ${res.unit}`;
 };
 
-export { formatSmidge, createError, getAbbreviatedText, getFormattedTimestamp, getAddress, formatBytes };
+const getNodeStatusText = (status) => {
```
Comment (IlyaVi) [inline app/infra/utils.js:60]: move it to Network.js as this is the single usage case

File: app/redux/node/reducer.js [hunk 7]
```diff
@@ -19,13 +20,55 @@ const initialState = {
   nodeIpAddress: nodeConsts.DEFAULT_URL
 };
 
+let startUpDelay = 0;
```
Comment (IlyaVi) [inline app/redux/node/reducer.js:23]: you can make it static in Network screen and pass to action as parameter
Anyway logic for ui display in single component doesn't look good in reducer. Better return it to component and use static fields for more bug protected code

File: app/screens/network/Network.js [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+
+type Props = {
+  node: Object,
+  status: Object,
+  nodeIndicator: Object,
+  history: RouterHistory
+};
+
+class Network extends Component<Props, State> {
+  startUpDelay = 5; // eslint-disable-line react/sort-comp
```
Comment (IlyaVi) [inline app/screens/network/Network.js:88]: this and next var are not in use

File: app/screens/network/Network.js (hunk 1 above)
Comment (Motzart) [reply app/screens/network/Network.js:109]: fix

File: app/screens/network/Network.js (hunk 2 above)
Comment (Motzart) [reply app/screens/network/Network.js:123]: fix

File: app/screens/network/Network.js (hunk 3 above)
Comment (Motzart) [reply app/screens/network/Network.js:158]: fix

File: app/screens/network/Network.js (hunk 5 above)
Comment (Motzart) [reply app/screens/network/Network.js:181]: fix

File: app/infra/utils.js (hunk 6 above)
Comment (Motzart) [reply app/infra/utils.js:60]: fix

File: app/redux/node/reducer.js (hunk 7 above)
Comment (Motzart) [reply app/redux/node/reducer.js]: fix

File: app/screens/network/Network.js (hunk 8 above)
Comment (Motzart) [reply app/screens/network/Network.js:88]: fix

**Comment by Motzart** [comment]
Screenshot 2020-10-22 at 11 15 23

**Review by IlyaVi — APPROVED** [review]

**Review by IlyaVi — APPROVED** [review]

**Pull request merged**